### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Add a resource via Azure Active Directory_Attach with Azure AD_ScreenReader: JAWS is not announcing the updated list item for the "Resource Type" combo box.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/AzureStorageExplorer#3467]
**GitHub Tags:** #C&AI_Aug2020_BMAzure Tools Storage Explorer; #A11yMAS; #A11YTCS; #DesktopApp; #Azure Tools Storage Explorer; #WCAG1.3.1; #Win32; #A11ySev2; #Benchmark; #JAWS;#FTP

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application Version: 1.14.2
Windows Version: Windows 10
Screen Reader : JAWS(2020.2006.12ILM)

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Add a resource via Azure Active Directory(Azure AD)" radio button and hit enter on Next button.
4. "Attach with azure AD" screen gets opened and navigate to Next button and hit enter.
5. Start JAWS and navigate using tab key.
6. Navigate till "Resource type" dropdown using tab key and change the selection of the list item using up/down arrow key.
7. Observe the announcement by JAWS.

### Actual:
After changing the selection of the list item of the "Resource Type" combo box, the JAWS is not announcing the updated list item.
This issue is observed for all the dropdown present in the application.
**This issue is not observed with NVDA.**
### Expected:
After changing the selection of the list item of the "Resource Type" combo box, the JAWS should announce the updated list item.

### User Impact:
Screen reader users won't understand which list item is selected.

### Attachment: 
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-shwarh_microsoft_com/EWnEiT7wVYNHtI87KEB_DNkBYAKqlEBEeKtmcYJSY6FRwg?e=Rfh6iF

**Comment by craxal** [comment]
@12-shweta Similar to #3470. Can you share a reference to the exact recommendation this issue is in violation of?

WCAG 1.3.1 defines recommendations for info and relationships. There is nothing in the recommendation that says a form control must announce an when it's contents are updated, especially when that control already has focus. I have not observed the behavior using test webpages with combo boxes.

**Comment by 12-shweta** [comment]
@craxal 
WCAG1.3.1 state that Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.
[3467](https://github.com/microsoft/AzureStorageExplorer/issues/3467) -> In this bug the issue is that after updating any list item present in the "Resource Type" combo box, the JAWS is not announcing updated list item that is screen reader is not announcing updated structure for the combo box.
 [3470](https://github.com/microsoft/AzureStorageExplorer/issues/3470) -> This bug is totally different. It is related to WCAG4.1.3 that is Status message.

**Comment by craxal** [comment]
@12-shweta As I mentioned above, #3467 is a know issue. We have identified this issue in our Accessibility Assessment documentation. Since NVDA doesn't have this problem, it is likely a problem with JAWS and not Storage Explorer.

I'm not sure why you mentioned #3470 here, as it's unrelated to this issue. I will add additional notes to that bug.

**Comment by 12-shweta** [comment]
As per the discussion with SME, closing this bug as a known issue.
